src/orange/detail/UnitInserter.cc

```diff
@@ -191,7 +172,8 @@ SimpleUnitId UnitInserter::operator()(UnitInput const& inp)
         // Store the bbox or an infinite bbox placeholder
         if (inp.volumes[i].bbox)
         {
-            bboxes.push_back(make_fast_bbox(inp.volumes[i].bbox));
+            bboxes.push_back(
+                calc_bumped<fast_real_type, real_type>(inp.volumes[i].bbox));
```

Comment:
The way we set up the function signature, the second template argument can be inferred:
```suggestion
            bboxes.push_back(calc_bumped<fast_real_type>(inp.volumes[i].bbox));
```